src/components/ApplicationForm/About/About.module.scss

```diff
@@ -0,0 +1,19 @@
+@use '~@ht6/react-ui/dist/styles/mixins.module';
+@use '~@ht6/react-ui/dist/styles/colors.module';
+@use '../../../styles/mixins.module' as localMixins;
+@use '../../../styles/vars.module';
+
+.field{
+    input{
+        border: none;
+        background-color: colors.getColor(neutral-400, null, 0.18);
+        color: colors.getColor(neutral-400);
+        &:disabled{
+            background-color: colors.getColor(neutral-50, null, 0.08);
+            color: colors.getColor(neutral-50);
+        }
+        &:focus{
+            color: colors.getColor(shades-100);
```

Comment:
Unsure what should be included here, not specified in the main design specification. Currently, it transitions to a white-background field, and as such a colour that is viewable was included; however, it is up to design what should be included.

Reply:
Changed behaviour to match landing page